src/main/java/ee/tuleva/onboarding/comparisons/ComparisonService.java

```diff
@@ -56,10 +60,11 @@ public Money compare(ComparisonCommand in, User user) throws IsinNotFoundExcepti
         Fund tulevaFundToCompareTo = fundRepository.findByIsin(in.getIsinTo());
         in.getManagementFeeRates().put(tulevaFundToCompareTo.getIsin(), tulevaFundToCompareTo.getManagementFeeRate());
 
-        Money averageSalary = averageSalaryService.getMyAverageSalary(user.getPersonalCode());
-        in.setMonthlyWage(averageSalary.getAmount());
+        if(in.monthlyWage == null) {
+            throw new ArgumentNullException("monthlyWage can not be null");
```

Comment:
Maybe use `IllegalArgumentException` instead of `com.rollbar.utilities.ArgumentNullException`